q2_american_gut/plugin_setup.py

```diff
@@ -28,22 +32,49 @@
     citation_text='https://doi.org/10.1101/277970'
 )
 
+plugin.register_formats(QiitaMetadataFormat, QiitaMetadataDirectoryFormat)
 
-def dummy(foo: biom.Table) -> biom.Table:
-    return foo
+plugin.register_semantic_types(QiitaMetadata)
+plugin.register_semantic_type_to_format(QiitaMetadata, \
+                                artifact_format=QiitaMetadataDirectoryFormat)
 
 
+# TODO: add support for shotgun retrieval
```

Comment:
Should these be issues vs TODOs on code?

Reply:
This seems to be the Q2 way... no preference